schema/adminschema/v1.json

```diff
@@ -100,6 +100,37 @@
           "$ref": "Client"
         }
       }
+    },
+    "Connector": {
+      "id": "Connector",
+      "type": "any",
+      "description": "A configuration object which describes how dex will federate identity. Because the actual structure differs for each kind of connector, the object here is unspecified."
```

Comment:
Can I add a link to https://github.com/coreos/dex/blob/master/Documentation/connectors-configuration.md? This seems like the canonical documentation on connectors.